Documentation/ceph-quickstart.md

```diff
@@ -71,6 +71,7 @@ The first step is to deploy the Rook operator. Check that you are using the [exa
 ```console
 cd cluster/examples/kubernetes/ceph
 kubectl create -f common.yaml
+kubectl create -f crds.yaml
```

Comment:
Ah thanks for fixing this, the tldr section was updated above, but this was missed. How about this single line instead?
```
kubectl create -f crds.yaml -f common.yaml -f operator.yaml
```

Reply:
@travisn how can we update our main website too?

Reply:
yes, working on it now